var/spack/repos/builtin/packages/py-pytest-benchmark/package.py

```diff
@@ -14,6 +14,11 @@ class PyPytestBenchmark(PythonPackage):
     pypi = "pytest-benchmark/pytest-benchmark-3.2.3.tar.gz"
 
     version("3.2.3", sha256="ad4314d093a3089701b24c80a05121994c7765ce373478c8f4ba8d23c9ba9528")
+    version(
+        "3.4.1",
+        sha256="40e263f912de5a81d891619032983557d62a3d85843f9a9f30b98baea0cd7b47",
+        preferred=True,
+    )
```

Comment:
Can you put the newer version first and remove the `preferred=True`? Spack automatically prefers the newest stable release